src/script/sign.cpp

```diff
@@ -302,6 +381,218 @@ struct Stacks
 };
 }
 
+// Iterates through all inputs of the partially signed transaction and just produces signatures for what it can and adds them to the psbt partial sigs
+bool SignPartialTransaction(PartiallySignedTransaction& psbt, const CKeyStore* keystore, int nHashType)
```

Comment:
`SignPartialTransaction` sounds after we have partial transactions. Suggest `SignPartialSignedTransaction`.

Reply:
Changed to `SignPartiallySignedTransaction`